### ISSUE TITLE
Skupper-docker delete fails to remove network if containers do not exist

### ISSUE THREAD

**Issue by dmc5179** [skupperproject/skupper-docker#4]
If the skupper-docker pods have already been deleted the skupper-docker delete command errors out and leaves the skupper-network in place, preventing a reinstall unless the network is manually remove. skupper-docker delete correctly removes the skupper-network if the pods are also there:

```
./skupper-docker delete
Stopping skupper proxy containers...
Stopping skupper-router container...
2020/03/31 22:04:03 Router container does not exist

[ec2-user@ip-172-31-65-204 skupper-docker]$ docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
187ffdc4d9cc        bridge              bridge              local
d96269db0576        host                host                local
6f63331ec14d        none                null                local
8e74cc85b8d6        skupper-network     bridge              local
```

**Comment by dmc5179** [comment]
When the containers do exist and delete is run:

```
./skupper-docker delete
Stopping skupper proxy containers...
Stopping skupper-router container...
Removing skupper-network network...
Skupper resources now removed

[ec2-user@ip-172-31-65-204 skupper-docker]$ docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
187ffdc4d9cc        bridge              bridge              local
d96269db0576        host                host                local
6f63331ec14d        none                null                local
```

**Comment by ajssmith** [comment]
Fixed in 0.2.0

**Comment by dmc5179** [comment]
@ajssmith Thanks!
